.github/workflows/pull_request_verification.yml

```diff
@@ -21,21 +20,25 @@ jobs:
       - name: Check correct commit message
         run: ./.github/scripts/commit/checkCommitMessage.sh
 
+  lint-check:
+    uses: ./.github/workflows/setup_docker_and_execute_command.yml
+    with:
+      command: ./gradlew lintDebug
+    secrets: inherit
+
+  ktlint-check:
+    uses: ./.github/workflows/setup_docker_and_execute_command.yml
+    with:
+      command: ./gradlew ktlintCheck
+
   build:
-    runs-on: ubuntu-latest
-    container: gradle:8.0.0-jdk17
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-      - name: Setup Android SDK
-        uses: android-actions/setup-android@v2.0.10
-      - name: Read google-service.json
-        env:
-          FIREBASE_GOOGLE_SERVICE: ${{ secrets.FIREBASE_GOOGLE_SERVICE }}
-        run: echo $FIREBASE_GOOGLE_SERVICE > app/src/google-services.json
-      - name: Check lint
-        run: ./gradlew lintDebug
-      - name: Check ktlint
-        run: ./gradlew ktlintCheck
-      - name: Build with Gradle
-        run: ./gradlew build
+    uses: ./.github/workflows/setup_docker_and_execute_command.yml
+    with:
+      command: ./gradlew assemble
+    secrets: inherit
+
+  tests:
+    uses: ./.github/workflows/setup_docker_and_execute_command.yml
```

Comment:
Please check if there is a possibility to reference only a name of the job instead of whole file

Reply:
I think it is not possible accordingly to this documentation [https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow](link) 

### From documentation: 
You reference reusable workflow files using one of the following syntaxes:

* `{owner}/{repo}/.github/workflows/{filename}@{ref}` for reusable workflows in public and private repositories.

* `./.github/workflows/{filename}` for reusable workflows in the same repository.

Reply:
Keep it as it is, we will later for some improvements :)